packages/backend/src/tests/propertyBased/arbitraries/arbitraryTableName.ts

```diff
@@ -0,0 +1,5 @@
+import { fc } from '@fast-check/vitest';
+
+export function arbitraryTableName(tableName: string): fc.Arbitrary<string> {
```

Comment:
This is kind-of a useless abstraction

arbitraryTableName(schema) could be useful, but this is essentially just a layer of indirection